plume-models/src/timeline/mod.rs

```diff
@@ -9,13 +9,14 @@ pub(crate) mod query;
 
 use self::query::{Kind, QueryError, TimelineQuery};
 
-#[derive(Clone, Queryable, Identifiable)]
+#[derive(Clone, Debug, PartialEq, Queryable, Identifiable, AsChangeset)]
 #[table_name = "timeline_definition"]
 pub struct Timeline {
     pub id: i32,
     pub user_id: Option<i32>,
     pub name: String,
     pub query: String,
+    pub ord: i32,
```

Comment:
What is this field for? Is it important to always have a defined order for timelines? And how should it be determined: by the user, or automatically?

Reply:
hum, I don't know, my through was that the first one would be on top, the second under and so on, but it clearly limit what can be done, I'll revert that